pkgs/unified_analytics/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 ## 5.8.6-wip
 
 - Refactored session handler class to use the last modified timestamp as the last ping value
+- Consolidate `Session` functionality into `UserProperty`
```

Comment:
Is this PR as a whole a breaking change?

Reply:
i think believe so, all of the public APIs we export are still the same

Reply:
Nevermind, we did add a new method `FakeAnalytics.sendPendingErrorEvents` to `FakeAnalytics` which we do export.

I think that would justify bumping to 6.0.0?

Reply:
i think so